pkg/expression/builtin_miscellaneous_vec.go

```diff
@@ -219,6 +220,122 @@ func (b *builtinUUIDSig) vecEvalString(ctx EvalContext, input *chunk.Chunk, resu
 	return nil
 }
 
+func (b *builtinUUIDv4Sig) vectorized() bool {
+	return true
+}
+
+func (b *builtinUUIDv4Sig) vecEvalString(ctx EvalContext, input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	result.ReserveString(n)
```

Comment:
it looks to me that the result of `uuid` function is fixed length? If yes, then we can do more precise space reservation. (ReserveString assume that each row has 8 bytes).

Reply:
Yes, `UUID()`, `UUID_V4()` and `UUID_V7()` are all fixed length.  A UUID is 36 characters in text format, and 16 bytes (128-bit) in binary format.

However, as the two new functions just do what the existing `UUID()` did, I think we should fix this outside of this PR.